fdbclient/MasterProxyInterface.h

```diff
@@ -67,10 +67,11 @@ struct MasterProxyInterface {
 	}
 
 	void initEndpoints() {
-		getConsistentReadVersion.getEndpoint(TaskPriority::ProxyGetConsistentReadVersion);
```

Comment:
This priority is used in non-proxy contexts now, so its name would probably warrant being changed if we intend to keep using it.

Reply:
@etschannen Don't forget the plan to change this to remove the `Proxy` prefix.